x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/spatial/SpatialGridLicenseTestCase.java

```diff
@@ -63,8 +63,8 @@ protected int precision() {
     /**
      * Test that the geo_grid functions are disabled outside of SNAPSHOT.
```

Comment:
nit: javadoc is stale.

I like this. At first I thought we might as well remove this test, but asserting that the capability is enabled is actually better. Guards against accidentally disabling the capability and not noticing that we broke prod.

Reply:
Fixed the stale comment. I also think I might refine this test a little more when supporting geo_shapes, and it might increase more in value then.